backend/src/middleware/handleHtmlContent/handleContentData.js

```diff
@@ -0,0 +1,69 @@
+import extractMentionedUsers from './notifications/extractMentionedUsers'
```

Comment:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-07-01T22:34:15Z" title="Tuesday, July 2nd 2019, 12:34:15 am +02:00">Jul 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,69 @@
+import extractMentionedUsers from './notifications/extractMentionedUsers'
```


is `extractMentionedUsers` in the right place `notifications` or should it rather be somewhere else?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-02T06:14:13Z" title="Tuesday, July 2nd 2019, 8:14:13 am +02:00">Jul 2, 2019</time>_

---


it’s only used for notifications. Therefore that should be okay. @ulfgebhardt